src/hrflow_connectors/connectors/poleemploi/schemas.py

```diff
@@ -0,0 +1,281 @@
+import datetime
+import typing as t
+from enum import Enum, IntEnum
+from typing import Any, List
+
+from pydantic import BaseModel, Field, validator
+
+POLEEMPLOI_REFERENCES_ENDPOINT = (
+    "https://api.emploi-store.fr/partenaire/offresdemploi/v2/referentiel/"
+)
+
+
+def validate_date(value: Any) -> Any:
+    try:
+        _ = datetime.date.fromisoformat(value)
+        return value
+    except ValueError:
+        raise ValueError(
+            f"Invalid time format given {value}, expected format ISO 8601 format"
+            " ISO-8601 (YYYY-MM-DDTHH:MM:SSZ)"
+        )
+
+
+class JobLocation(BaseModel):
+    libelle: str
+    latitude: float
+    longitude: float
+    codepostal: str
+    commune: str = Field(
+        description=(
+            "INSEE code of the commune"
+            "A GET request for the list of accepted choices from the Offres"
+            " d'emploi API"
+            "to this endpoint :"
+            f" {POLEEMPLOI_REFERENCES_ENDPOINT}communes"
+        ),
+    )
+
+
+class Entreprise(BaseModel):
+    nom: t.Optional[str]
+    description: t.Optional[str]
+    logo: t.Optional[str]
+    url: t.Optional[str]
+    entrepriseAdaptee: bool
+
+
+class Partenaire(BaseModel):
+    nom: str
+    url: str
+    logo: str
+
+
+class OrigineOffreTag(IntEnum):
+    poleEmploi = 1
+    partenaire = 2
+
+
+class OrigineOffre(BaseModel):
+    origine: OrigineOffreTag
+    urlOrigine: t.Optional[str]
+    partenaires: List[Partenaire]
+
+
+class ExperienceExige(str, Enum):
+    d = "D"  # Débutant accepté
+    s = "S"  # Expérience souhaitée
+    e = "E"  # Expérience exigée
```

Comment:
Explicite is better than implicite, modifie les variable a une lettre par le nom complet